### ISSUE TITLE
icmp from userspace applications breaks on Debian

### ISSUE THREAD

**Issue by isi-lincoln** [rcgoodfellow/raven#14]
Ubuntu/Debian spams `Error listening for ICMP packets: socket: permission denied` due to userspace permissions for icmp handling.

```
lthurlow@ubuntu:~/go/src/github.com/rcgoodfellow/raven/models/2net$ ./launch.sh 

Fetching walrus~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Cloning into 'walrustf'...
remote: Counting objects: 151, done.
remote: Total 151 (delta 0), reused 0 (delta 0), pack-reused 151
Receiving objects: 100% (151/151), 164.49 KiB | 0 bytes/s, done.
Resolving deltas: 100% (58/58), done.
Checking connectivity... done.

Building~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

clearing out any artifacts from previous runs
building system
Attempting copy from: https://mirror.deterlab.net/rvn/img/debian-stretch to: /var/rvn/img/debian-stretch
Attempting copy from: https://mirror.deterlab.net/rvn/img/cumulusvx-3.5 to: /var/rvn/img/cumulusvx-3.5

Deploying~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

launching vms
waiting for vms to come on network
Error listening for ICMP packets: socket: permission denied
Error listening for ICMP packets: socket: permission denied
Error listening for ICMP packets: socket: permission denied
```

**Comment by hugelgupf** [comment]
Just jumping in here cause it's in my emails now... 

ICMP sockets are either raw sockets or dgram sockets with IPPROTO_ICMP. Raw sockets are only accessible if you have CAP_NET_RAW. So the program that tries to use ICMP needs to run as root.

Tl;Dr - missing setuid bit on ping?

**Comment by rcgoodfellow** [comment]
This [pr](https://github.com/rcgoodfellow/raven/pull/15) may fix this. I think it's a sysctl issue. Raven runs as root by default to use the system level libvirt.

**Comment by rcgoodfellow** [comment]
Fixed in [15](https://github.com/rcgoodfellow/raven/pull/15)
